src/.eslintrc

```diff
@@ -0,0 +1,11 @@
+---
+extends: ../node_modules/builder-docs-archetype/config/eslint/.eslintrc-react
```

Comment:
I believe so? The config was unable to be found using just `node_modules/` as a base